### PR TITLE
Update deps

### PR DESCRIPTION
**Pull request by dawnerd**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,14 +4,14 @@ Server code should follow [this styleguide](https://github.com/airbnb/javascript
 
 ### Installation
 
-`npm install --save-dev eslint-config-firstandthird eslint-config-airbnb@8.0.0 eslint-plugin-import eslint-plugin-react eslint-plugin-jsx-a11y eslint@"^2.9.0"`
+`npm install --save-dev eslint-config-firstandthird eslint-config-airbnb@10.0.1 eslint@^2.9.0 eslint-plugin-jsx-a11y@^1.2.0 eslint-plugin-import@^1.7.0 eslint-plugin-react@^5.0.1`
```
Comment (jgallen23) [inline README.md]: eslint version needs updating here too.

**Comment by jgallen23** [comment]
what changed in airbnb?  wondering if we should

**Comment by dawnerd** [comment]
Probably a smart idea to clone it anyways. They have a ton of rules and it's hard to tell what's different than the defaults. I feel like a lot of our code would fall under the default eslint config pretty well. It's what I have setup on a couple side projects and haven't really noticed a difference.

This PR was more to get rid of the npm warnings.

**Comment by jgallen23** [comment]
just update readme and then you can merge and release

**Pull request merged**